src/plugin/kubearmor.go

```diff
@@ -432,6 +456,26 @@ func StartKubeArmorRelay(StopChan chan struct{}, cfg types.ConfigKubeArmorRelay)
 					Type:          res.Type,
 				}
 
+				if len(nsFilter) > 0 {
+					for _, ns := range nsNotFilter {
+						if !strings.Contains(res.NamespaceName, ns) {
+							continue
+						}
+					}
+				} else if len(nsNotFilter) > 0 {
+					for _, notns := range nsNotFilter {
+						if strings.Contains(res.NamespaceName, notns) {
+							continue
+						}
+					}
+				}
+
+				for _, srcFilter := range fromSourceFilter {
+					if strings.Contains(res.Source, srcFilter) {
+						continue
+					}
+				}
+
```

Comment:
This code seems redundant to what is available before. Can you refactor and create a function to handle this?